docs/book/src/tasks/healthcheck.md

```diff
@@ -16,15 +16,16 @@ at the bottom of this page for full details of MachineHealthCheck limitations.
 
 ## What is a MachineHealthCheck?
 
-A MachineHealthCheck is a resource within the Cluster API which allows users to define conditions under which Machines within a Cluster should be considered unhealthy.
+A MachineHealthCheck is a resource within the Cluster API which allows users to define conditions under which Machines within a Cluster should be considered unhealthy. 
+A MachineHealthCheck is defined on a management cluster and scoped to a particular workload cluster.
 
-When defining a MachineHealthCheck, users specify a timeout for each of the conditions that they define to check on the Machine's Node,
+When defining a MachineHealthCheck, users specify a timeout for each of the conditions that they define to check on the Machine's Node;
 if any of these conditions is met for the duration of the timeout, the Machine will be remediated.
-The action of remediating a Machine should trigger a new Machine to be created, to replace the failed one.
+The action of remediating a Machine should trigger a new Machine to be created to replace the failed one.
 
 ## Creating a MachineHealthCheck
 
-Use the following example as a basis for creating a MachineHealthCheck:
+Use the following example as a basis for creating a MachineHealthCheck for worker nodes:
```

Comment:
Are you planning to add also an example for control-plane node remediation?

Reply:
There is an example for KCP node remediation in the document now as part of this PR. (Should start on/around line 57 or so.)